src/decimal.rs

```diff
@@ -1097,27 +1103,33 @@ fn sub_internal(value: &mut [u32], by: &[u32]) -> u32 {
     overflow
 }
 
+//#[inline]
 fn sub_part(left: u32, right: u32, overflow: u32) -> (u32, u32) {
```

Comment:
Perhaps you could inline always this and reuse above?

Reply:
Actually, if we use `sub_part` in `sub3_internal`, it reduce performance about 12%. I suppose, we get speed down because of additional transforms `u32` values in `u64` and vice versa.